classes/Content/class.xonoContentGUI.php

```diff
@@ -1,60 +1,76 @@
 <?php
 
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
 use ILIAS\DI\Container;
 use ILIAS\HTTP\Wrapper\WrapperFactory;
+use ILIAS\Plugin\OnlyOffice\Enum\PluginAsset;
+use ILIAS\Plugin\OnlyOffice\Form\PluginConfigForm;
+use ILIAS\Plugin\OnlyOffice\Repository;
 use ILIAS\Refinery\Factory;
-use srag\DIC\OnlyOffice\DIC\DICInterface;
-use srag\DIC\OnlyOffice\DICStatic;
-use srag\Plugins\OnlyOffice\InfoService\InfoService;
-use srag\Plugins\OnlyOffice\StorageService\DTO\FileVersion;
-use srag\Plugins\OnlyOffice\StorageService\Infrastructure\File\ilDBFileChangeRepository;
-use srag\Plugins\OnlyOffice\StorageService\Infrastructure\File\ilDBFileRepository;
-use srag\Plugins\OnlyOffice\StorageService\Infrastructure\File\ilDBFileVersionRepository;
-use srag\Plugins\OnlyOffice\StorageService\StorageService;
-use srag\Plugins\OnlyOffice\Utils\DateFetcher;
-use srag\Plugins\OnlyOffice\Utils\OnlyOfficeTrait;
+use ILIAS\Plugin\OnlyOffice\StorageService\DTO\FileVersion;
+use ILIAS\Plugin\OnlyOffice\StorageService\Infrastructure\File\ilDBFileChangeRepository;
+use ILIAS\Plugin\OnlyOffice\StorageService\Infrastructure\File\ilDBFileRepository;
+use ILIAS\Plugin\OnlyOffice\StorageService\Infrastructure\File\ilDBFileVersionRepository;
+use ILIAS\Plugin\OnlyOffice\StorageService\StorageService;
+use ILIAS\Plugin\OnlyOffice\Utils\DateFetcher;
 
 class xonoContentGUI extends xonoAbstractGUI
 {
-    use OnlyOfficeTrait;
-
     public const BASE_URL = ILIAS_HTTP_PATH;
 
     protected ilOnlyOfficePlugin $plugin;
     protected StorageService $storage_service;
     protected int $file_id;
-    private $tpl;
+    private ilGlobalTemplateInterface $tpl;
 
     public const CMD_STANDARD = 'showVersions';
     public const CMD_SHOW_VERSIONS = 'showVersions';
     public const CMD_DOWNLOAD = 'downloadFileVersion';
     public const CMD_EDIT = xonoEditorGUI::CMD_EDIT;
     private Factory $refinery;
     private WrapperFactory $httpWrapper;
+    private Repository $repo;
 
     public function __construct(
-        Container          $dic,
+        Container $dic,
         ilOnlyOfficePlugin $plugin,
-        int                $object_id
-    )
-    {
+        int $object_id
+    ) {
         global $DIC;
```

Comment:
Since you get `dic` through the constructor, you should not access `global $DIC`